src/azure-openapi-linter/rules/ArraySchemaMustHaveItems.ts

```diff
@@ -12,19 +12,14 @@ rules.push({
   name: ArraySchemaMustHaveItems,
   severity: "warning",
   category: "SDKViolation",
-  mergeState: MergeStates.individual,
+  mergeState: MergeStates.composed,
```

Comment:
why should this one be composed instead of individual? just trying to understand the difference in this case.